internal/lib/lib.go

```diff
@@ -46,6 +46,12 @@ var shardSizeMap = map[string]uint32{
 	"SMALL":  1,
 }
 
+var fqdnEnum = map[string]int32{
+	"<svc>.<ns>.<subdomain>": 1,
+	"<svc>-<ns>.<subdomain>": 2,
+	"false":                  3,
```

Comment:
Do we need the "false" key/value here, since we are anyway returning 3 (in `GetL4FqdnFormat`) in case of no match?

Reply:
Done.